kedro-databricks-bootcamp/02_databricks/rocketfuel/notebooks/02_2-Kedro-on-Databricks-part-2.ipynb

```diff
@@ -72,14 +77,67 @@
     "%load_ext kedro.ipython"
    ]
   },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "Ensure your project now contains the databricks configuration created in the previous notebook. The content should be as follows:\n",
+    "\n",
+    "```yaml\n",
+    "_uc_catalog: #your catalog location\n",
+    "_uc_schema: #your schema location\n",
+    "\n",
+    "companies_raw:\n",
+    "  type: spark.SparkDataset\n",
+    "  filepath: /Volumes/${_uc_catalog}/${_uc_schema}/bronze/companies.csv\n",
+    "  file_format: csv\n",
+    "  load_args:\n",
+    "    header: True\n",
+    "    inferSchema: True\n",
+    "\n",
+    "reviews_raw:\n",
+    "  type: spark.SparkDataset\n",
+    "  filepath: /Volumes/${_uc_catalog}/${_uc_schema}/bronze/reviews.csv\n",
+    "  file_format: csv\n",
+    "  load_args:\n",
+    "    header: True\n",
+    "    inferSchema: True\n",
+    "\n",
+    "shuttles_raw:\n",
+    "  type: pandas.ExcelDataset\n",
+    "  filepath: /Volumes/${_uc_catalog}/${_uc_schema}/bronze/shuttles.xlsx\n",
```

Comment:
I was facing an issue with this. `spark.SparkDataset` understands the `/Volume` syntax, but I don't think `pandas.ExcelDataset` does. Did it work for you?

And yet, since we're running this locally, maybe we can use the local filepath in this case?